### PR TITLE
2-D view of the CRV

### PR DESCRIPTION
**Pull request by NamithaChitrazee**
2-D view tabs of the CRV added. The tabs appear only when "showCRV" is set to true.

More updates to follow.

### CODE REVIEW

File: src/REveMainWindow.cc [hunk 1]
```diff
@@ -111,6 +115,10 @@ void REveMainWindow::showNodesByName(TGeoNode* n, const std::string& str, bool o
                   FrontTracker_gdmltag = j;
                   
                  }
+                std::string CRVstr("CRS"); // ("CRSScintillatorBar_1_0");
```
Comment (sophiemiddleton) [inline src/REveMainWindow.cc:118]: can you fix the indents?

File: src/REveMainWindow.cc [hunk 2]
```diff
@@ -111,6 +115,10 @@ void REveMainWindow::showNodesByName(TGeoNode* n, const std::string& str, bool o
...
+                if (name.find(CRVstr)!= std::string::npos){
```
Comment (sophiemiddleton) [inline src/REveMainWindow.cc:119]: I added something similar, can you reuse the bool I made?

Comment (NamithaChitrazee) [reply src/REveMainWindow.cc:119]: Hi Sophie, yes I think I can.

Comment (NamithaChitrazee) [reply src/REveMainWindow.cc]: Hi Sophie, it's fixed.

**Comment by sophiemiddleton** [comment]
How is this progressing Namitha? Let me know when it's ready for review

**Comment by NamithaChitrazee** [comment]
Hi sophie, will check this today.

**Comment by brownd1978** [comment]
Is this PR still under development?

**Comment by sophiemiddleton** [comment]
I will chat with Namitha tomorrow to decide if this is still needed. I think what we had here was only part of the plan

**Pull request closed without merging**